### DISCUSSION TITLE
Attempting to Use SubTrees with this fork of Git

### DISCUSSION THREAD

**Discussion by tomcdevoe** [git-for-windows/git#3302] [Q&A] [answered]
I'm relatively new to Git and have been digging into it in the last few weeks. We would like to use a hierarchical approach to repos. I've looked at submodules which to me seem to be geared to referenced class libraries that are used by one or more application projects. I read articles about using subtrees and this sounded like a great idea. I'm using the latest version of this fork of Git and attempting to use the subtree approach and get "It looks like either your git installation or your
git-subtree installation is broken." I looked in the C:\Program Files\Git\mingw64\libexec\git-core folder and find only a readme type of file but no corresponding exe such as the other items in this folder in this folder that have both a readme and exe.

Is it possible to use subtrees with this fork of Git?
Thanks.

**Comment by rimrul** [comment, marked as answer]
Yes, it's usually possible to use `git subtree` in Git for Windows. The issue you're encountering is that an update to `git subtree` in Git broke `git subtree` on windows and made it into Git v2.32.0 and by extension into Git for Windows 2.32.0. There is a proposed fix in #3264 and gitgitgadget#978 that's currently being discussed on the mailing list. You can test it using the installer artifact of https://github.com/git-for-windows/git/actions/runs/922160854.

**Reply by tomcdevoe** [reply]
I ran the installer as part of the zip at the link you provided and subtree functionality is now available. Thank you so much for your help!
